vpdq/cpp/io/vpdqio.cpp

```diff
@@ -149,6 +153,23 @@ bool readVideoResolution(
   width = videoParameter->width;
   return true;
 }
+
+bool readVideoDuration(
+    const string& inputVideoFileName, int& duration, const char* programName) {
+  AVFormatContext* pFormatCtx = avformat_alloc_context();
+  int rc =
+      avformat_open_input(&pFormatCtx, inputVideoFileName.c_str(), NULL, NULL);
+  if (rc != 0) {
+    fprintf(
+        stderr,
+        "%s: could not open video \"%s\".\n",
+        programName,
+        inputVideoFileName.c_str());
+    return false;
+  }
+  duration = pFormatCtx->duration / 1000000;
```

Comment:
can you have 1000000 be a const value that describes the unit conversion it is doing?